config.json

```diff
@@ -3,7 +3,11 @@
     "url": "http://engineering.underdog.io",
     "name": "Underdog.io Engineering blog",
     "owner": "Underdog.io Engineering",
-    "description": "Articles and insights from the Underdog.io Engineering team"
+    "description": "Articles and insights from the Underdog.io Engineering team",
+    "gaId": {
+      "production": "UA-49880770-1",
+      "development": "UA-64967323-1"
```

Comment:
Then we'd have `locals[node_env].gaId` and probably be easier to manage in the future should we add more prod/dev specific config settings.